Resolves #28 Adds JS click handlers to show/hide meal details on mobile

liamwooding opened the pull request.

liamwooding committed:
Resolves #28 Adds JS click handlers to show/hide meal details on mobile

richsilv commented on client/meals/meals.js:
```diff
@@ -0,0 +1,7 @@
+Template.meals.events = {
+  'click .meal': function (e) {
+    console.log(e)
```
Can you remove the logging?

richsilv commented:
The problem is that if you click on a meal on desktop, it then removes the overlay until you click on another one, which looks a bit odd when the hover effects are still working as well:

![makelunch](https://cloud.githubusercontent.com/assets/3180526/9656507/b1637d18-5230-11e5-99b2-1da05df312af.png)

Can we have it so that it fires when you click on the overlay?  That way you'll only end up firing the event on mobile as the overlay will disappear on hover on desktop.

liamwooding committed:
Removes logging

liamwooding committed:
Click listener on meals attaches to tint rather than meal

liamwooding committed:
Moves meals LESS into its own file

liamwooding committed:
Merge branch 'master' into past-meals-hover

liamwooding committed:
Adds responsive width detector divs;

Meals visibility only toggled at mobile width;
Meals visibility is reset when resizing back up to SM width

liamwooding commented:
@richsilv :+1:

richsilv committed:
Debounce window resize handler
